data-jdbc/src/test/groovy/io/micronaut/data/jdbc/oraclexe/OracleXERepositorySpec.groovy

```diff
@@ -296,4 +297,94 @@ class OracleXERepositorySpec extends AbstractRepositorySpec implements OracleTes
         faceRepository.delete(face)
     }
 
+    void "test insert returning book"() {
```

Comment:
Custom queries using `@Query` with `RETURNING INTO` not supported like for Postgres since not able to gather out parameters in `RawQueryMethodMatcher`